examples/pytorch/question-answering/run_seq2seq_qa.py

```diff
@@ -476,10 +483,11 @@ def preprocess_validation_function(examples):
             max_length=max_seq_length,
             padding=padding,
             truncation=True,
-            return_overflowing_tokens=True,
             return_offsets_mapping=True,
+            return_overflowing_tokens=True,
         )
-        # Tokenize targets with the `text_target` keyword argument
+
+        # Setup the tokenizer for targets
```

Comment:
The comment should be the one deleted in the diff and not the one added :-)

Reply:
sorry, revert to original version